week2/internal/domain/model/film.go

```diff
@@ -0,0 +1,5 @@
+package model
+
+type FilmRequest struct {
```

Comment:
not sure what is this struct for

Reply:
Me either, I just figured we might need it later. Removed :v